STYLE-PRACTICES.md

```diff
@@ -285,28 +237,28 @@ override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
 
 ### **不推荐**在视图层以外调用界面方法。
 
-视图层（如 `Activity` 和 `Fragment`）理应承担所有视图任务，如显示动画、显示对话框、显示上下文菜单、跳转到新页面等等。其他层，尤其是 `ViewModel`，不应当执行任何有关方法。
+视图层（如 `Compose`）理应承担所有视图任务，如显示动画、显示对话框、显示上下文菜单、跳转到新页面等等。其他层，尤其是 `ViewModel`，不应当执行任何有关方法。
 
 > **例外**
 >
-> 在自定义的视图-控制器一体类（如 `Feature`）中，`startActivity` 是可容忍的。
+> 在自定义的视图-控制器一体类（如 `Feature`）中，`navigate` 是可容忍的。
 
 ## 应用惯例
 
-### **不要**过多地在通用类中使用只针对复旦 UIS 账号的实现。
+### **要**针对不同账户系统使用不同的数据类型。
 
-旦夕是面向多类型账户系统的，不应想当然地把 `PersonInfo` 视作只包含 UIS 账号密码的数据类型，也不要在设置中过多出现「复旦」有关的字样。
+旦夕是面向多类型账户系统的，`PersonInfo` 是只针对 UIS 账号密码的数据类型。
```

Comment:
Is this sentence contradicting itself? Why does `PersonInfo` only contain Fudan's UIS information if we are aiming at a general-use account? Are there other `PersonInfo`-like data classes in the future? Clarify it.

Reply:
There should not be any general-use account. UIS account `UISInfo` and FDUHole account `OTJWTToken` should be kept separate since they can be logged in and logged out independently. Also, any data class like `PersonInfo`, which I guess means detailed info of an account, should load from network every time app open. Only those info binding to an account token or credential, like `id`, `name`, can be stored in persistent storage.

I will update this sentence.